src/World.h

```diff
@@ -708,6 +725,9 @@ class cWorld :
 	/** Queues a task onto the tick thread, with the specified delay. */
 	void ScheduleTask(int a_DelayTicks, cTaskPtr a_Task);
 
+	/** Queues a task onto the tick thread, with the specified delay. */
+	void cWorld::ScheduleLambdaTask(int a_DelayTicks, std::function<void(cWorld&)> a_Func);
```

Comment:
The indentation is visual studio formatting. I'll dig around the settings to see if I can disable it.

Ah, cWorld was left from me moving it around, It compiled fine for me, but I'll remove it.

I tried creating an overload to ScheduleTask, but the compiler complained the call was ambiguous, I'm too new to C++ to understand why the typedef cTaskPtr could be implicitly cast to std::function...